tests/validation_tests/test_failures.py

```diff
@@ -435,28 +435,40 @@ def test_failures_to_messages():
 def test_group_validation_messages():
     data = [
         # A - combine these
-        ("Project Admin", "Project Details", "A1", "You left cells blank."),
-        ("Project Admin", "Project Details", "A2", "You left cells blank."),
+        ("Project Admin", "Project Details", "A1", "TownsFundRoundFourValidationFailure", "You left cells blank."),
+        ("Project Admin", "Project Details", "A2", "TownsFundRoundFourValidationFailure", "You left cells blank."),
         # B - combine these
-        ("Project Admin", "Programme Details", "D4", "You left cells blank."),
-        ("Project Admin", "Programme Details", "D4, D5, D7", "You left cells blank."),
+        ("Project Admin", "Programme Details", "D4", "TownsFundRoundFourValidationFailure", "You left cells blank."),
+        (
+            "Project Admin",
+            "Programme Details",
+            "D4, D5, D7",
+            "TownsFundRoundFourValidationFailure",
+            "You left cells blank.",
+        ),
         # C - do not combine these due to different sections
-        ("Risk Register", "Project Risks - Project 1", "G24", "Select from the dropdown."),
-        ("Risk Register", "Project Risks - Project 2", "G43", "Select from the dropdown."),
+        ("Risk Register", "Project Risks - Project 1", "G24", "WrongInputFailure", "Select from the dropdown."),
+        ("Risk Register", "Project Risks - Project 2", "G43", "WrongInputFailure", "Select from the dropdown."),
         # D - do not combine these due to different descriptions
-        ("Outcomes", "Programme-level Outcomes", "E5", "You left cells blank."),
-        ("Outcomes", "Programme-level Outcomes", "E7", "Select from the dropdown."),
+        ("Outcomes", "Programme-level Outcomes", "E5", "WrongInputFailure", "You left cells blank."),
+        ("Outcomes", "Programme-level Outcomes", "E7", "WrongInputFailure", "Select from the dropdown."),
```

Comment:
Looks like your tests are failing because the `error_type` needs to be the last value in the tuple, with `message` as the penultimate one